<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UILiveLink</class>
 <widget class="QWidget" name="UILiveLink">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>166</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LiveLink Tracker Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../gui/images/opentrack.png</normaloff>../gui/images/opentrack.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinPortNumber">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Add to axis</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>yaw</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="add_yaw">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-180</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>pitch</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>roll</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="add_pitch">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-180</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="add_roll">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-180</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>startEngineClicked()</slot>
  <slot>stopEngineClicked()</slot>
  <slot>cameraSettingsClicked()</slot>
 </slots>
</ui>
